src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,48 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인하는 테스트")
+    void setTest1() {
+        assertThat(numbers).hasSize(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    @DisplayName("Set의 contains() 메소드를 활용해 1, 2, 3의 값이 존재하는지 테스트")
+    void setTest2(int number) {
+        assertThat(numbers.contains(number)).isTrue();
+    }
```

Comment:
```suggestion
    void setTest2(int number) {
        assertThat(numbers).contains(number);
    }
```
이런 메소드도 활용할 수도 있으니 참고만 해주세요!!